group_vars/openstack/defaults.yml

```diff
@@ -1,10 +1,10 @@
 ---
 
 cluster_image_prefix: "centos-7-"
-cluster_deploy_user: "root"
+cluster_deploy_user: "centos"
 
 # When making the volume that will become the cluster image, use this minimum size
-cluster_image_volume_min_size: 5
+cluster_image_volume_min_size: 8
```

Comment:
The latest centos 7 cloud image seems to need more than 5 GB, I tried both 6 and 7 but that wasn't enough.